test/test_cli.py

```diff
@@ -320,7 +320,7 @@ def test_unknown_warning(self, caplog):
         assert f"png was detected with version UNKNOWN in file {filename}" in warnings
 
     @pytest.mark.skipif(
-        LONG_TESTS() == 1, reason="broken in long tests for reasons unknown"
+        LONG_TESTS() > 0, reason="broken in long tests for reasons unknown"
```

Comment:
I'm curious why `> 0` worked but `== 1` didn't work for us in the CI

Reply:
Pretty sure it's a type conversion issue with the LONG_TESTS environment variable going from str to int or something.  Might be worth seeing if we can get LONG_TESTS to return a bool or something instead of what we're doing now to make it more clear.

Reply:
I've filed #2404 so maybe we can make it better if we want

Reply:
@BreadGenie @terriko 
After some snooping around in https://github.com/intel/cve-bin-tool/pull/2411#discussion_r1040085179 I can say that this change (from `== 1` to `> 0`) didn't do anything. The test ran simply because `LONG_TESTS` was set to '0'.

P.S. Why it failed is a different story! (I don't know why, haven't looked into it)